src/scroll-timeline-css-parser.js

```diff
@@ -94,6 +100,16 @@ export class StyleParser {
     return null;
   }
 
+  getViewTimelineOptions(timelineName) {
```

Comment:
return { subject: element }

Then no need for extra downstream processing.  Simply call the new ViewTimeline.